src/main/java/org/jenkinsci/plugins/DependencyCheck/transformer/FindingsTransformer.java

```diff
@@ -131,6 +131,14 @@ public JSONObject transform(List<Finding> findings) {
         vulnReferences.put("filterable", false);
         columns.add(vulnReferences);
 
+        final JSONObject projReferences = new JSONObject();
+        projReferences.put("name", "dependency.projectReferences");
+        projReferences.put("title", "Project References");
```

Comment:
Use "Referenced In Projects/Scopes" as title, scope is the part I did not expect in the name of the project. The title should advise about that (as html report does)

Reply:
OK